### PR TITLE
Add new CRD scale workload

### PR DESCRIPTION
**Pull request by smalleni**
This commit adds a new crd-scale workload under the ocp wrapper and also an example config file to run it standalone with the kube-burner command.
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [X] Documentation Update

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Comment by smalleni** [comment]
@vishnuchalla any thoughts?

**Review by vishnuchalla — APPROVED** [review]
lgtm

File: cmd/kube-burner/ocp-config/crd-scale/crd-scale.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  gc: {{.GC}}
+  indexerConfig:
+    esServers: ["{{.ES_SERVER}}"]
+    insecureSkipVerify: true
+    defaultIndex: {{.ES_INDEX}}
+    type: {{.INDEXING_TYPE}}
+jobs:
+  - name: crd-scale
+    namespace: crd-scale
+    jobIterations: 50
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/crd-scale/crd-scale.yml:12]: You need to use a var here

File: cmd/kube-burner/ocp-config/crd-scale/crd-scale.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    qps: 10
+    burst: 10
```
[the remark this answers is not in the document]
Comment (smalleni) [reply cmd/kube-burner/ocp-config/crd-scale/crd-scale.yml:14]: yeh typo i guess...will fix that.

[the remark this answers is not in the document]
Comment (smalleni) [reply cmd/kube-burner/ocp-config/crd-scale/crd-scale.yml]: fixed

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Pull request merged**